### ISSUE TITLE
Constraints does not work

### ISSUE THREAD

**Issue by armandojimenez** [RatelHub/rflutter_alert#110]
When I use the constraints param, it does not take effect and it still takes the content width, looks weird on tablets

**Comment by armandojimenez** [comment]
Anyone know how to deal with this?

**Comment by alihalabyah** [comment]
+1

**Comment by ibrahimdevs** [comment]
@armandojimenez can you share your constraints added code?

**Comment by armandojimenez** [comment]
constraints: const BoxConstraints(maxWidth: 400),

Nothing I try works

**Comment by AsjadSiddiqui** [comment]
I was able to fix this by playing around with alertPadding in AlertStyle
BoxConstraints width parameters still don't work, but the alert was able to fit full device width.
You can control the width using alertPadding parameter in AlertStyle
```
Alert(
      context: context,
      style: AlertStyle(
        alertPadding: EdgeInsets.symmetric(horizontal: 20),

        // minWidth doesn't work
        constraints: BoxConstraints(
          minHeight: 0.6 * height,
          minWidth: 0.9 * width,
        ),
      ),
    );
```

**Comment by armandojimenez** [comment]
> I was able to fix this by playing around with alertPadding in AlertStyle BoxConstraints width parameters still don't work, but the alert was able to fit full device width. You can control the width using alertPadding parameter in AlertStyle
> 
> ```
> Alert(
>       context: context,
>       style: AlertStyle(
>         alertPadding: EdgeInsets.symmetric(horizontal: 20),
> 
>         // minWidth doesn't work
>         constraints: BoxConstraints(
>           minHeight: 0.6 * height,
>           minWidth: 0.9 * width,
>         ),
>       ),
>     );
> ```

With this the dialog does not full expand to the content width?

**Comment by armandojimenez** [comment]
The problem is with tablets, boxConstraints does not work for max height and width

**Comment by armandojimenez** [comment]
Is this going to be worked on? Its easy reproducible, the constraint param does not work, add a long text to the description param and use a tablet, the dialog will extend the width of the text

**Comment by armandojimenez** [comment]
![IMG_0002](https://user-images.githubusercontent.com/28992210/146452102-377d1da6-95f8-4e09-a84b-20c465f445eb.PNG)

Example, the dialog is taking the width of the string passed to desc param, also happens with the title.

my constrain code:
`constraints: const BoxConstraints(maxWidth: 400),`

Nothing works in terms of param.

**Comment by robertoltrocha** [comment]
Hi,

I have the same issue. BoxContrains is not working

**Comment by ibrahimdevs** [comment]
@armandojimenez this will be fixed in next version: V2.0.6
